calico-enterprise_versioned_docs/version-3.19-2/release-notes/index.mdx

```diff
@@ -225,3 +225,16 @@ This release includes bug fixes and improvements.
 * Security updates.
 
 To update an existing installation of Calico Enterprise 3.19, see [Install a patch release](../getting-started/manifest-archive.mdx).
+
+### Calico Enterprise 3.19.4 bug fix release
+
+November 7, 2024
+
+Calico Enterprise 3.19.4 is now available.
+This release includes bug fixes and improvements.
+
+#### Bug fixes
+
+TBD
```

Comment:
- Fixed an issue where excessive API calls were made by the operator if users are running a large number of Egress Gateways.
- Security updates.

Reply:
Added.